### ISSUE TITLE
Trending collection snapping on narrow devices?

### ISSUE THREAD

**Issue by defagos** [SRGSSR/playsrg-ios#44]
When scrolling through trending medias, it is difficult to manually stop so that the media thumbnail is not cut on the left or on the right.

`UICollectionView` provides a native snap mechanism to improve the user experience in such cases (without having use paging, which makes navigation slow).

**Comment by defagos** [comment]
As with Apple iOS TV app, I implemented snapping on all rows. Since we have very few items per row, I also used fast deceleration so that navigating by items is made a bit slower. Finally, I ensured snapping reveals whether items are located before in a row.

Both these measures provide a more convenient way to browse the new content. The user avoids micro-managing the scrolling velocity to navigate in each row, we ensure that cells are not cut by device boundaries, and that grid layouts are homogeneous with aligned items.

The code is ready for review on the `feature/swimlane-snapping` branch.
